scripts/Muon/GUI/FrequencyDomainAnalysis/FFT/fft_model.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import (absolute_import, division, print_function)
+
+from six import iteritems
+
+import mantid.simpleapi as mantid
+
+
+class FFTWrapper(object):
+
+    """
+    A class to wrap the different parts
+    of the FFT and its preprocessing.
+    This keeps the main FFT class simple.
+    """
+
+    def __init__(self, FFT):
+        self.name = "FFT"
+        self.model = FFT
+
+    def cancel(self):
+        self.model.cancel()
+
+    def loadData(self, inputs):
+        """
+        store the data in the wrapper for later
+        """
+        if "phaseTable" in inputs:
+            self.phaseTable = inputs["phaseTable"]
+        else:
+            self.phaseTable = None
+        if "preRe" in inputs:
+            self.preRe = inputs["preRe"]
+        else:
+            self.preRe = None
+        if "preIm" in inputs:
+            self.preIm = inputs["preIm"]
+        else:
+            self.preIm = None
+        if "FFT" in inputs:
+            self.FFT = inputs["FFT"]
+        else:
+            self.FFT = None
+        self.model.setRun(inputs["Run"])
+
+    def execute(self):
+        """
+        runs the relevant parts of the FFT and the preprocessing
+        """
+        if self.phaseTable is not None:
```

Comment:
Is this just the same as `if self.phaseTable`?

Reply:
some OS can be fussy, so its safer

Reply:
This style is actually recommended by the Python PEP-8 style guide: https://www.python.org/dev/peps/pep-0008/#programming-recommendations for comparisons with singleton objects. I don't think it should be OS specific...